src/renderer/Theming.js

```diff
@@ -0,0 +1,77 @@
+const React = require('react')
+const StyledThemeProvider = require('styled-components').ThemeProvider
+const { EventEmitter } = require('events')
+
+const defaultTheme = Object.freeze({
+  deltaPrimaryBg: '#415e6b',
+  deltaPrimaryBgLight: '#4c6e7d',
+  deltaPrimaryFg: '#ffffff',
+  deltaPrimaryFgLight: '#d0d0d0',
+  deltaHover: '#ececec',
+  deltaChatPrimaryFg: '#070c14',
+  deltaChatPrimaryFgLight: '#62656a',
+  deltaChatMessageBubbleSelf: '#efffde',
+  deltaChatMessageBubbleOther: '#ffffff',
+  deltaInfoMessageBubbleBg: '#000000',
+  deltaInfoMessageBubbleColor: 'white',
+  deltaChatMessageBubbleSelfStatusColor: '#4caf50',
+  deltaFocusBlue: '#42A5F5',
+  deltaSelected: '#4c6e7d' // same as deltaPrimaryBgLight
+})
+
+class ThemeManager extends EventEmitter {
+  constructor () {
+    super()
+    const themeJSON = window.localStorage.getItem('theme')
+    this.currentTheme = themeJSON !== null ? JSON.parse(themeJSON) : {}
+
+    window.ThemeManager = this // only for using fron the dev console
+  }
+
+  setTheme (theme) {
+    window.localStorage.setItem('theme', JSON.stringify(theme))
```

Comment:
Can we use rc for this? This way you can set the theme via the config.ini file

Reply:
I like the idea to use a config file and/or theme file. What about a steam like aproach? having a theme folder where users can add theme files. that then get displayed in an theme dropdown in the settings. But on the other hand I like the option to create your theme from the the dev console so maybe a combination of both methods... Or just a load from file option in the front end like upload a `theme.yaml` file to import it. And download to export it...

Reply:
I would prefer having it in the config folder and not in some browser/electron storage (as an upload/download would suggest). But for the moment it's not about making customized themes but integrating a dark theme.

Reply:
> But for the moment it's not about making customized themes but integrating a dark theme.

Why not set the foundation now? we don't have to show the option to the user. I'll see in how far it makes sense. I mean we don't need a theme editor now for example. But a folder with theme files from that you get a drop down which the user can select from in the settings and maybe the option to change it from the dev console seems doable and not too much out of scope...